joint_0_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_0
    joint_speed: 1.17
    motor:
        id: 10
        init: 550
        min: 270
        max: 840

joint_1_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_1
    joint_speed: 1.17
    motor:
        id: 11
        init: 590
        min: 300
        max: 900

joint_2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_2
    joint_speed: 1.17
    motor:
        id: 12
        init: 512
        min: 0
        max: 1023


joint_3_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_3
    joint_speed: 1.17
    motor:
        id: 13
        init: 512
        min: 0
        max: 1023

joint_4_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_4
    joint_speed: 1.17
    motor:
        id: 14
        init: 512
        min: 0
        max: 1023
